examples/browser/wdio.base.conf.js

```diff
@@ -1,9 +1,14 @@
 // @ts-check 
 const http = require('http');
 const path = require('path');
+const temp = require('temp');
 
 const wdioRunnerScript = require.resolve('webdriverio/build/lib/runner.js');
 
+// Remove .track() if you'd like to keep the workspace and other temporary
+// files after running the tests.
+const temptrack = temp.track();
```

Comment:
Keeping the workspaces by default might be the best option. Won't ```temp``` create them in the system's temp directory, to be wiped-out automatically eventually (on reboot on Linux)?

Reply:
In my experience, it's good practice to have the tests clean up after themselves.  Yes, /tmp gets cleaned on reboot, but some servers don't get restarted for months/years, and dozens of people run things that put stuff in /tmp. So I think it's better if things don't leave things in /tmp after they are done using it.

If we really need to keep it, I would create the temporary workspace at a defined location under the Theia build directory, which would be wiped and re-used the next time you run the tests.

Reply:
Ok to keep like this.

If we need to troubleshoot a test failure, and need inspecting the workspace for anything suspicious, I guess we do as the comment says and re-run to get a new copy of the workspace that remains.